src/app/static/src/app/components/plots/choropleth/Choropleth.vue

```diff
@@ -21,7 +21,7 @@
                              :level-labels="featureLevels"
                              @detail-changed="onDetailChange"
                              @indicator-changed="onIndicatorChange"></map-control>
-                <map-legend :metadata="colorIndicator"
+                <map-legend v-show="!emptyFeature" :metadata="colorIndicator"
```

Comment:
I'd probably just do do this as a v-if, no reason to render the control at all here.

Reply:
The v-if behaviour was erratic on the map, freezing indicators control, while v-show happens to work well.